docs/tutorials/tutorial-sketches-theta.md

```diff
@@ -274,7 +202,7 @@ SELECT THETA_SKETCH_ESTIMATE(
 FROM ts_tutorial
 ```
 
-![Count distinct with Theta sketches, filters, and set operations](../assets/tutorial-theta-09.png)
```

Comment:
Why did you delete these three images?

![image](https://github.com/user-attachments/assets/f4f060b9-4dc6-4357-a0b4-81cbbd8d95a7)

Reply:
Resolved